### ISSUE TITLE
The name tf.GraphDef is deprecated.

### ISSUE THREAD

**Issue by ghost** [canaydogan/nudity#1]
When using class nudity = Nudity(): "The name tf.GraphDef is deprecated. Please use tf.compat.v1.GraphDef instead."

`nudity = Nudity()`
`WARNING: Logging before flag parsing goes to stderr.
W0701 11:45:44.926168  9768 deprecation_wrapper.py:119] From C:\Users\...\Python37\lib\site-packages\nudity\__init__.py:51: The name tf.GraphDef is deprecated. Please use tf.compat.v1.GraphDef instead.`

**Comment by minimalisticMe** [comment]
Did you get it to run?

**Comment by ghost** [comment]
Yes, but I removed what was deprecated. Here's my fork: https://github.com/Sam-Marx/nudity

**Comment by micodev** [comment]
do a pull request dude.

**Comment by ThisLimn0** [comment]
Jesus Christ.
